src/components/HTMLParser/utils.tsx

```diff
@@ -0,0 +1,67 @@
+import { DOMNode, domToReact, attributesToProps, HTMLReactParserOptions } from 'html-react-parser';
+import { Text, TextVariants } from '../Text';
+import { Link, LinkVariants } from '../Link';
+
+export const options = {
+  replace({ attribs, children, name }: { attribs: Record<string, string>; children: React.ReactNode; name: string }) {
+    if (name === 'h1') {
+      return (
+        <Text variant={TextVariants.display1}>
+          {domToReact(children as DOMNode[], options as HTMLReactParserOptions)}
+        </Text>
+      );
+    }
+
+    if (name === 'h2') {
+      return (
+        <Text variant={TextVariants.heading2}>
+          {domToReact(children as DOMNode[], options as HTMLReactParserOptions)}
+        </Text>
+      );
+    }
+
+    if (name === 'h3') {
+      return (
+        <Text variant={TextVariants.heading3}>
+          {domToReact(children as DOMNode[], options as HTMLReactParserOptions)}
+        </Text>
+      );
+    }
+
+    if (name === 'h4') {
+      return (
+        <Text variant={TextVariants.heading4}>
+          {domToReact(children as DOMNode[], options as HTMLReactParserOptions)}
+        </Text>
+      );
+    }
+
+    if (name === 'blockquote') {
+      return (
+        <Text variant={TextVariants.blockquote}>
+          {domToReact(children as DOMNode[], options as HTMLReactParserOptions)}
+        </Text>
+      );
+    }
+
+    if (name === 'p') {
+      return (
+        <Text variant={TextVariants.body1}>{domToReact(children as DOMNode[], options as HTMLReactParserOptions)}</Text>
+      );
+    }
+
+    if (name === 'a') {
+      // why do I have to do this?
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { class: className, ...rest } = attribs;
```

Comment:
Anyone know why typescript insisted that I rename the class variable here?

Reply:
@davidicus because you're not using the `className` variable anywhere. You can rename it to `_className` to indicate that you intended to strip it out (this pattern is in our eslint-rule) of the destructured attributes if that's what you really want.